src/features/application-search/ApplicationSearchBar.tsx

```diff
@@ -56,7 +56,10 @@ const getSearchQuerystring = () => {
   if (window.location.search.startsWith('?q=')) {
     const terms = window.location.search.split('=');
     if (terms.length === 2) {
-      return decodeURI(terms[1]);
+      // Only need to decode if terms[1] is a valid encoding
+      try {
+        return decodeURI(terms[1]);
+      } catch (err) { /* do nothing */ }
```

Comment:
Did some research to find a better way to determine if a string can be decoded. None of the options I found were without edge cases and many suggested to use the try catch method.